tests/manage/z_cluster/test_hugepages.py

```diff
@@ -0,0 +1,99 @@
+import logging
+import pytest
+
+from ocs_ci.helpers.sanity_helpers import Sanity
+from ocs_ci.ocs import constants
+from ocs_ci.utility.utils import (
+    enable_huge_pages,
+    disable_huge_pages,
+)
+from ocs_ci.ocs.node import (
+    wait_for_nodes_status,
+    get_nodes,
+)
+from ocs_ci.ocs.resources.pod import (
+    wait_for_pods_to_be_running,
+)
+from ocs_ci.framework.testlib import (
+    bugzilla,
+    skipif_ocs_version,
+    skipif_external_mode,
+    ignore_leftovers,
+    E2ETest,
+    tier2,
+)
+
+log = logging.getLogger(__name__)
+
+
+@tier2
+@skipif_external_mode
+@skipif_ocs_version("<4.8")
+@bugzilla("1995271")
+@bugzilla("1995271")
+@pytest.mark.polarion_id("OCS-2754")
+@ignore_leftovers
+class TestHugePages(E2ETest):
+    """
+    Enable huge pages post cluster deployment
```

Comment:
Please make it clear that in this test, huge pages is enabled post ODF installation to avoid confusion as huge pages can also be enabled immediate after ocp cluster deployment.

Reply:
updated